### DISCUSSION TITLE
Consumption history

### DISCUSSION THREAD

**Discussion by Thomsus** [motoz/PellMon#116] [Q&A]
Hi, is there a way to see pellet consumption older than one year?

I can only see consumption within the last year, using the URL x.x.x.x:8081/consumptionview/flotconsumption1y

**Comment by motoz** [comment]
Well, no but yes, sort of. With the default settings for the rrd database there is about 15 years of data retained, but I never got around to making a ui for showing more than one year. But in that way there is a way, someone just have to write some code to show it...

**Reply by motoz** [reply]
The current code is here https://github.com/motoz/PellMon/blob/master/src/Pellmonsrv/plugins/consumption/__init__.py
